README.md

```diff
@@ -1,40 +1,96 @@
+DCF contains core scripts and styles which are used as 'plumbing' and a foundation to build a theme. It is created to be brand agnostic.
+The core scripts and styles help create a more consistent user experience, incorporate the best possible web accessibility affordances, and allow for ease of development across themes, while individual themes built upon it allows for customizations and brand identity.
 
-### required theme directories and files
 
-* The `/theme/name/css` directory is where generated css will be placed.
-* The `/theme/name/js` directory is where generated js will be placed.
 
-## Installing the project
+## Folder Structure 
+The DCF core uses Gulp as its build tool for task automation, supoorts SASS and ES6 JS syntax. DCF core also employs 'cutting the mustard' approach to using JS browser feature detection to request for fallbacks that most modern evergreen browsers have already taken care of. This is a more user friendly and performant approach to providing fallbacks only when it is needed. 
 
-Run `npm install` to install the required node modules.
 
-You are good to go!
 
-## Building a theme
+### Directories and Files 
+Source files should be placed in the `assets/src` folder and running the default gulp task will output the production-ready files in the `assets/dist` folder. Developers should only have to work with `assets/src` and `assets/dist` folder. 
+
+* The `assets/src` directory contains the core scripts and files for DCF.
+* The `assets/build` directory contains temporary files between build processes (e.g. concatenated JS files before minification).
+* The `assets/dist` directory contains production ready scripts and files for DCF.
+
+
+#### Sass Files
+`/assets/src/scss/**/*.scss`
+
+There will be no SCSS compilation `.scss` files in DCF Core. Sass file compilations is done on the theme layer to allow additional theme `.scss` files to be compiled alongside core `.scss` files. This also allows theme layer scss files to make use of includes and mixins defined in DCF core. There should be two "main/index" scss files in the _theme_ layer:
+* `main.scss` - brings together all the core and theme layer sass partials for compilation
+* `mustard.scss` - brings together partials for fallback styles that will be used in the 'cutting the mustard' approach 
+
+
+#### JavaScript Files
+`/assets/src/js`
+
+* There are three folders under `/assets/src/js`:
+    * `vendor`: Contains third-party vendor JS files that are required by DCF, since vendor files do not change frequently, outputting a separate vendor file helps with browser caching.
+    * `mustard`: Contains third-party vendor JS polyfills that will be pulled in if browser doesn't cut the mustard
+    * `app`: Custom written JS that includes common distributed and optional files. Supports ES6 syntax which will be transpiled by Babel during the build process.
+      *`common`: Custom written JS that is needed by DCF and will be packaged with and distributed with core
```

Comment:
Indentation appears to be different here. Seems to be inconsistent in other places in this document too. Might break the markdown formatting.

Reply:
Good catch. Fixed